code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -106,6 +106,55 @@
 		if(!Org.is_robotic())
 			Org.rejuvenate()
 
+/datum/reagent/medicine/tran_acid
+	name = "Tranexamic acid"
+	id = "tran_acid"
+	description = "A specialised chemical used for the treatment of truamatic bleeding and to control bleeding during surgery."
+	reagent_state = LIQUID
+	color = "#fcec96"
+	overdose_threshold = 40
+	taste_description = "time running out"
+// This reagent's effects are handled in handle_blood() code
+
+/datum/reagent/medicine/calzeo
+	name = "Calicum Zeolite"
+	id = "calzeo"
+	description = "A naturally occuring mineral used for treating mass hemorrhage. Use with caution."
+	reagent_state = LIQUID
+	color = "#f3d7a4"
+	overdose_threshold = 15
+	harmless = FALSE
+	taste_description = "Bonemeal"
+// This reagent's effects are handled in handle_blood() code
+
+/datum/reagent/medicine/calzeo/overdose_process(mob/living/M, severity)
+	var/list/overdose_info = ..()
+	var/effect = overdose_info[REAGENT_OVERDOSE_EFFECT]
+	var/update_flags = overdose_info[REAGENT_OVERDOSE_FLAGS]
+	if(severity == 1)
+		if(effect <= 1)
+			M.visible_message("<span class='warning'>[M] suddenly and violently vomits!</span>")
+			M.fakevomit(no_text = 1)
```

Comment:
`no_text = TRUE`
Same below. (You did apply the change via github, but that was subsequently lost due to a force-push)

Reply:
this would make it inconsistent with the rest of the reagents, not just medicine.dm and rest of the reagents would have to be updated to reflect that and that would be beyond the scope of the RP